### ISSUE TITLE
Implement vis_histogram

### ISSUE THREAD

**Issue by jrosell** [ropensci/visdat#163]
When doing EDA one of the most common tasks is to get sense of the distribution of the values for each of the variables.

Using a new function vis_histogram would I expect to get a grid of histogram plots for each variable in the dataset.
I think that the function arguments could be used to specify the shared properties for all plots, or could be specified by variable.

I think something like that could work:

```r
library(visdat)
(vis_histogram(airquality))
(vis_histogram(airquality, bins = 30, options = list(Month = list(bins = 12)))
```

The previous code should show two plots. The first one showing an histogram for each of the variables using defaults of geom_histogram. And the second one should be similar but the Month variable would be 12 bins.

Are you interested in this kind of feature?

**Comment by njtierney** [comment]
That sounds awesome! I've got some other features that I would like to work on first but if you'd like to open a PR with some starting code I'm more than happy to provide feedback and contribute via that route  - what do you think?

**Comment by jrosell** [comment]
@njtierney would you like me to create a pull request for the master branch or for a new vis-histogram branch?

**Comment by njtierney** [comment]
Hi @jrosell !

a pull request for master would be great - if you can entitle it something like vis-histogram?

**Comment by jrosell** [comment]
You can see my pull request. I know it doesn't implement customization of multiple aesthetics differently, but I think vis_histogram can do what is expected.

**Comment by matthewingmire** [comment]
Hi! Is vis_histogram available for public consumption?

**Comment by jrosell** [comment]
Not yet. You can see my pull request from my for here: https://github.com/jrosell/visdat/commit/bb66dda24f944916a81f0ce506287c4a26fac527

**Comment by matthewingmire** [comment]
Thanks for the quick reply.On Mar 9, 2023, at 8:30 AM, Jordi ***@***.***> wrote:﻿
Not yet. You can see my pull request from my for here: ***@***.***

**Comment by njtierney** [comment]
This is now implemented, thanks to #164 - thanks to @jrosell :)
